async/index.browser.js

```diff
@@ -7,36 +7,25 @@ var crypto = self.crypto || self.msCrypto
 // Despite the fact the source code is quite long, its entropy
 // is low and there are lots of duplicates - just what compressors
 // like GZIP and Brotli likes the best.
-var i
-var url = '_-' + String.fromCharCode(
-  // ASCII codes for 0...9
-  i = 48, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1,
+var i = 36
+var url = '-_'
 
-  // ASCII codes for A...Z
-  i += 8, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1,
-
-  // ASCII codes for a...z
-  i += 7, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1
-)
+while (i--) {
+  url += i.toString(36)
+  i > 9 && (url += i.toString(36).toUpperCase())
+}
 
 module.exports = function (size) {
-  size = size || 21
   var id = ''
-  var bytes = crypto.getRandomValues(new Uint8Array(size))
+  size = size || 21
+
   // Compact alternative for `for (var i = 0; i < size; i++)`
   while (size--) {
     // We can’t use bytes bigger than the alphabet. 63 is 00111111 bitmask.
     // This mask reduces random byte 0-255 to 0-63 values.
     // There is no need in `|| ''` and `* 1.6` hacks in here,
     // because bitmask trim bytes exact to alphabet size.
-    id += url[bytes[size] & 63]
+    id += url[crypto.getRandomValues(new Uint8Array(1)) & 63]
```

Comment:
This change will kill performance since the single call of `getRandomValues` is much faster than multiple calls.

Reply:
![image](https://user-images.githubusercontent.com/6529684/73334212-2f4ad500-427d-11ea-8042-9e1c7ab6c5c8.png)